concepts/access_control.md

```diff
@@ -0,0 +1,72 @@
+# OpenTDF Access Control Concepts (ABAC)
+
+OpenTDF implements a sophisticated access control model known as **Attribute-Based Access Control (ABAC)**. This approach provides fine-grained, flexible, and scalable authorization for data protection.
+
+## What is ABAC?
+
+Attribute-Based Access Control is a security paradigm where access rights are granted based on the evaluated **attributes** of entities involved in an access request, rather than solely on roles or explicit permissions lists. Key components include:
+
+*   **Subject Attributes:** Characteristics of the entity requesting access (e.g., user's clearance, department, nationality, group memberships).
+*   **Resource Attributes:** Characteristics of the data or resource being accessed (e.g., data classification, sensitivity level, project code).
+*   **Policies:** Rules that define allowable actions by comparing subject, resource, and potentially environment attributes (e.g., "Allow access if subject's clearance >= resource's classification AND subject is in department 'X'").
+*   **Policy Enforcement Point (PEP):** The logical component that evaluates the policies against the attributes and makes the final access decision (grant or deny).
+
+ABAC offers significant advantages over traditional models, enabling dynamic access decisions that adapt to changing conditions without needing constant updates to user roles or access control lists (ACLs).
+
+## OpenTDF's Implementation of ABAC
+
+OpenTDF embeds ABAC directly into the data's protection layer. Here's how its components map to ABAC concepts:
+
+*   **Subject Attributes:** Represented by **Entity Entitlements**. These are the attribute instances asserted by the identity system or client about the user/entity requesting access. They are provided *to* the PEP during an access request.
+*   **Resource Attributes:** Defined within the TDF's [Policy Object](../schema/OpenTDF/policy.md) in the `dataAttributes` array. These specify the attribute instances *required* to access this specific piece of data.
+*   **Policies:** Defined by the combination of:
+    *   The `dataAttributes` required by the specific TDF.
+    *   The optional `dissem` list in the TDF's [Policy Object](../schema/OpenTDF/policy.md) acting as an initial filter.
```

Comment:
just checking that we want to talk about dissem, maybe people will want to use it, but wanted to make sure

Reply:
I believe we want to keep it in